pages/Magic-Workbench.md

```diff
@@ -0,0 +1,11 @@
+The Magic Workbench is a [Basic Machine](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Basic-Machine) in Slimefun.<br>
+It is used to craft many [magical items](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Magical-Items), [gadgets](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Magical-Gadgets) and even [armor](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Magical-Armor).
+
+## Obtaining
+The Magic Workbench is a Multiblock structure which consists of a Bookshelf, Crafting Table and Dispenser placed side by side.<br>
+A complete Magic Workbench looks like this :
+
+![Magic Workbench setup](https://raw.githubusercontent.com/TheBusyBiscuit/Slimefun4-Wiki/images/multiblock-magic-workbench.png)
+
+## Usage
+Place the items in the correct pattern in the Dispenser, then right-click the Crafting Table. If the recipe is correct, your item will appear in the Dispenser.
```

Comment:
You should also add the note about the Output Chest to this machine.
Like it is written on other machine pages too.